ios/RNZendeskChatModule.m

```diff
@@ -201,4 +201,23 @@ - (void) dismissChatUI {
 	});
 }
 
+RCT_EXPORT_METHOD(areAgentsOnline:
+	(RCTPromiseResolveBlock) resolve
+	rejecter: (RCTPromiseRejectBlock) reject) {
+
+  [ZDKChat.accountProvider getAccount:^(ZDKChatAccount *account, NSError *error) {
+		if (account) {
```

Comment:
If there is no account, this promise never resolves.

I'm going to add a code-snippet to reject in that situation.